### PR TITLE
Displays backer and sponsor count

### PR DESCRIPTION
**Pull request by carlosascari**
When a yearly income variable exists, it will also show the amount donated per year

### CODE REVIEW

File: frontend/src/components/public_group/PublicGroupHero.js [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
 import PublicTopBarV2 from '../../containers/PublicTopBarV2';
+import formatCurrency from '../../lib/format_currency';
 
 export default class PublicGroupHero extends React.Component {
+
+  renderHeroStatistics()
+  {
+    const { group } = this.props;
+    if (!group.backers.length) return;
+    const backerCount = group.backers.reduce((prev, curr) => prev + (curr.role === 'BACKER'), 0);
```
Comment (xdamman) [inline frontend/src/components/public_group/PublicGroupHero.js]: complicated way to compute this.
why don't you use `filterCollection(group.backers, {tier: tier.name})` with `import filterCollection from '../../lib/filter_collection';`

File: frontend/src/components/public_group/PublicGroupHero.js [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+    const sponsorCount = group.backers.reduce((prev, curr) => prev + (curr.role === 'SPONSOR'), 0);
```
Comment (xdamman) [inline frontend/src/components/public_group/PublicGroupHero.js]: there is no role "SPONSOR", it's a tier (see previous comment)

File: frontend/src/components/public_group/PublicGroupHero.js [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
 
 export default class PublicGroupHero extends React.Component {
 
   renderHeroStatistics()
   {
     const { group } = this.props;
     if (!group.backers.length) return;
-    const backerCount = group.backers.reduce((prev, curr) => prev + (curr.role === 'BACKER'), 0);
-    const sponsorCount = group.backers.reduce((prev, curr) => prev + (curr.role === 'SPONSOR'), 0);
+    const backerCount = filterCollection(group.backers, {tier: 'backer'}).length;
```
Comment (carlosascari) [inline frontend/src/components/public_group/PublicGroupHero.js]: This looks right, right? I am getting null for the `tier` property

**Comment by xdamman** [comment]
I have no idea how I can add a commit to this PR (I don't have access)

So I've just created a new PR with this: https://github.com/OpenCollective/website/pull/191

**Pull request closed without merging**